src/Avalonia.Controls/AppBuilder.cs

```diff
@@ -15,38 +16,53 @@ public abstract class AppBuilderBase<AppBuilder> where AppBuilder : AppBuilderBa
         /// <summary>
         /// Gets or sets the <see cref="IRuntimePlatform"/> instance.
         /// </summary>
-        public IRuntimePlatform RuntimePlatform { get; set; }
+        public IRuntimePlatform RuntimePlatform { get; private set; }
```

Comment:
Why did you changed that to `private set`? It's perfectly valid for a user to change runtime platform after calling constructor.